install_config/install/prerequisites.adoc

```diff
@@ -106,6 +106,11 @@ back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
 |===
 
+[IMPORTANT]
+====
+OpenShift supports only x86_64 servers.
+====
```

Comment:
Use product-title macro. Also, I recommend expansion to quantify what x86_64 is with "architecture":

{product-title} only supports servers with x86_64 architecture.